ea_restaurant_data_loader/test/lib/service/app_client_service_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule EaRestaurantDataLoader.AppClientServiceTest do
+  use EaRestaurantDataLoader.RepoCase
+
+  alias EaRestaurantDataLoader.AppClientService
+  alias EaRestaurantDataLoader.AppClientFixture
+  alias EaRestaurantDataLoader.UserFixture
+
+  describe "app client service test" do
+    test "get app client by client id should return expected app client" do
+      user_returned = UserFixture.build_and_insert_user("test-user", "test-username")
+
+      app_client_returned =
```

Comment:
app_client instead of app_client_returned

Reply:
Done.